public/docs/ts/latest/tutorial/toh-pt1.jade

```diff
@@ -32,164 +33,143 @@ include ../_quickstart_repo
+  then initialize it with an id of `1` and the name "Windstorm."
 
 +makeExample('toh-1/ts/app/app.component.ts', 'hero-property-1', 'app.component.ts (hero property)')(format=".")
 
 :marked
-  Because we changed the hero from a string to an object,
-  we update the binding in the template to refer to the hero’s `name` property.
+  Because you changed the hero from a string to an object,
+  update the binding in the template to refer to the hero's `name` property.
 
 +makeExample('toh-1/ts-snippets/app.component.snippets.pt1.ts', 'show-hero-2')
 :marked
-  The browser refreshes and continues to display our hero’s name.
-
-  ### Adding more HTML
-  Displaying a name is good, but we want to see all of our hero’s properties.
-  We’ll add a `<div>` for our hero’s `id` property and another `<div>` for our hero’s `name`.
-
-+makeExample('toh-1/ts-snippets/app.component.snippets.pt1.ts', 'show-hero-properties')
-:marked
-  Uh oh, our template string is getting long. We better take care of that to avoid the risk of making a typo in the template.
+  The browser refreshes and continues to display the hero's name.
 
-  ### Multi-line template strings
+  ### Adding HTML with multi-line template strings
 
-  We could make a more readable template with string concatenation
-  but that gets ugly fast, it is harder to read, and
-  it is easy to make a spelling error. Instead,
-  let’s take advantage of the template strings feature
-  in ES2015 and TypeScript to maintain our sanity.
-
-  Change the quotes around the template to back-ticks and
-  put the `<h1>`, `<h2>` and `<div>` elements on their own lines.
+  Displaying a name is good, but we want to see all of the hero's properties.
+  For this update, you'll add a `<div>` for the hero's `id` property and another `<div>` for the hero's `name`.
+  To keep the template readable, you'll make some additional changes using the template strings feature
+  in ES2015 and TypeScript:
+  change the quotes around the template to backticks
+  and put the `<h1>`, `<h2>`, and `<div>` elements on their own lines.
 
 +makeExample('toh-1/ts-snippets/app.component.snippets.pt1.ts', 'multi-line-strings', 'app.component.ts (AppComponent\'s template)')
 
-  .callout.is-important
-    header A back-tick is not a single quote
-    :marked
-      **Be careful!** A back-tick (`) looks a lot like a single quote (').
-      It's actually a completely different character.
-      Back-ticks can do more than demarcate a string.
-      Here we use them in a limited way to spread the template over multiple lines.
-      Everything between the back-ticks at the beginning and end of the template
-      is part of a single template string.
+// CF: I removed the note about backticks. It wasn't appearing in the browser and Naomi didn't think it was necessary.
 
 .l-main-section
 :marked
-  ## Editing Our Hero
+  ## Editing the hero name
 
   We want to be able to edit the hero name in a textbox.
```

Comment:
I'd get rid of "We want to be able to" and make it more direct. I might even delete the sentence altogether and use the idea to introduce the next sentence. 

"To edit the hero name in a textbox, refactor..."